compiler/damlc/daml-prim-src/DA/Exception/ContractError.daml

```diff
@@ -1,24 +0,0 @@
--- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
--- SPDX-License-Identifier: Apache-2.0
-
-{-# LANGUAGE CPP #-}
```

Comment:
same here we still have this exception in Daml.

Reply:
reverted